### PR TITLE
Fixed custom status issues

### PR DESCRIPTION
**Pull request by manojmalik20**
#### Summary

- Fixed MM-34924 Status placeholder in profile popover is not correctly themed
- Fixed MM-35649 Bots show the integration owner's custom status in post header in RHS

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-34924
- https://mattermost.atlassian.net/browse/MM-35649

#### Release Note

```release-note
Fixed MM-34924 and MM-35649 issues
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @manojmalik20!

File: components/post_profile_picture/post_profile_picture.tsx [hunk 1]
```diff
@@ -119,6 +119,9 @@ export default class PostProfilePicture extends React.PureComponent<Props> {
                 username={user?.username}
                 overwriteIcon={this.props.overwriteIcon}
                 overwriteName={overwriteName}
+                isBot={user && user.is_bot}
```
Comment (agarciamontoro) [inline components/post_profile_picture/post_profile_picture.tsx:122]: Nit: we could simplify this a bit
```suggestion
                isBot={user?.is_bot}
```

**Review by agarciamontoro — APPROVED** [review]
Looks great! Thank you, @manojmalik20 :tada:

I left a non-blocking comment below.

**Comment by agarciamontoro** [comment]
Thank you @manojmalik20!

All yours, @ogi-m :)

**Comment by agarciamontoro** [comment]
@ogi-m, I'm not sure how to fix the test server creation. Any idea?

**Comment by manojmalik20** [comment]
@agarciamontoro I guess you could check the logs. Maybe there's some info on why it's failing.

**Comment by ogi-m** [comment]
> I'm not sure how to fix the test server creation. Any idea?

Updating the branch usually does the trick @agarciamontoro @manojmalik20

**Comment by ogi-m** [comment]
It's probably because of the failing checks

**Comment by agarciamontoro** [comment]
@manojmalik20, thank you for your patience here :)

@ogi-m, good for QA!

**Review by ogi-m — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**